README.md

```diff
@@ -1,2 +1,3 @@
 # CarParking-System
 This is my first application.
+Thi is the 2nd line.
```

Comment:
Please do check for spelling mistakes @akashELXSI

Reply:
@developer-harishkt  FIXED.